qgis-app/plugins/validator.py

```diff
@@ -243,6 +244,14 @@ def validator(package):
                 "Cannot find a folder inside the compressed package: this does not seems a valid plugin"
             )
         )
+    # Check if package_name is PEP 8 compliant
+    if is_new and not re.match(r"^[a-z_][a-z0-9_]*$", package_name):
```

Comment:
@Xpirix can we use this instead? https://docs.python.org/3/library/stdtypes.html#str.isidentifier

Reply:
Yeah, it's simple than regex. Fixed it, thanks for the hint.